opensearchpy/_async/client/ingest.py

```diff
@@ -117,7 +117,7 @@ async def simulate(self, body, id=None, params=None, headers=None):
         :arg body: The simulate definition
         :arg id: Pipeline ID.
         :arg verbose: Verbose mode. Display data output for each
-            processor in executed pipeline.
+            processor in executed pipeline. (default: false)
```

Comment:
Nit: when we update templates we can say "Default is false." so it reads better.